package.json

```
@@ -11,7 +11,8 @@
     "prepare": "husky install",
     "func:ext-install": "func extensions install",
     "start": "func start",
-    "generate:issuer-api-models": "gen-api-models --api-spec ./openapi-issuer.yaml --out-dir ./src/infra/api-models"
+    "generate:issuer-api-models": "gen-api-models --api-spec ./openapi-issuer.yaml --out-dir ./src/infra/api-models",
+    "generate:tokenizer-api-models": "curl -s https://api.tokenizer.pdv.pagopa.it/docs/tokenizerpapis/openapi.json -o ./openapi-tokenizer.json; json2yaml ./openapi-tokenizer.json > ./openapi-tokenizer.yaml; gen-api-models --api-spec ./openapi-tokenizer.yaml --out-dir ./src/infra/tokenizer/models --client"
```

```suggestion
    "generate:tokenizer-api-models": "curl -s https://api.tokenizer.pdv.pagopa.it/docs/tokenizerpapis/openapi.json -o ./openapi-tokenizer.json; json2yaml ./openapi-tokenizer.json > ./openapi-tokenizer.yaml; gen-api-models --api-spec ./openapi-tokenizer.yaml --out-dir ./src/infra/tokenizer/models --request-types  --response-decoders --client"
```
You need to generate also `--request-types`  and `--response-decoders` to use client!